src/lib/rules/husky/apply-husky-configuration.ts

```diff
@@ -2,9 +2,9 @@ import { Rule, Tree } from '@angular-devkit/schematics';
 import { createHuskyRc } from './create-husky-rc';
 import { updateHuskyRc } from './update-husky-rc';
 
-export function applyHuskyConfiguration(): Rule {
+export function applyHuskyConfiguration(newHook: [string, string]): Rule {
```

Comment:
I don't like this one, because the parameter is ignored when .huskyrc doesn't exist. Any suggestions?

Reply:
For the moment I would just add a JSDoc comment here.
We won't need this anymore if templates can overwrite existing files.

Reply:
added a JSDoc